lib/remarkable_ruby/item.rb

```diff
@@ -0,0 +1,66 @@
+module RemarkableRuby
+  class Item
+    attr_reader :uuid, :path, :message, :success, :blob_url_get, :bookmarked, 
+      :blob_url_get_expires, :modified_client, :type, :current_page,
+      :connection
+
+    attr_accessor :parent, :name, :version
+
+    def initialize(attrs: nil, client: nil, path: nil)
+      @client = client ? client : Client.new
+      @connection = @client.connection
+      @path = path
+      @name = File.basename(path) if path
+
+      attrs.nil? ? init_from_defaults : init_from_attributes(attrs)
+    end
+
+#     def inspect
```

Comment:
Right now the return value for methods that return Folder/Documents are really long because the Faraday connection gets saved as an instance variable which is an object that itself has a ton of instance variables. So I think we should overwrite the Object#inspect method with _something_

Reply:
Gotcha. Do you want to create an issue to implement update inspect method and do that in a separate PR?

Reply:
Ok, yea. You're right, this PR already touches on too many topics.